org.librarysimplified.audiobook.views/src/main/java/org/librarysimplified/audiobook/views/PlayerTOCAdapter.kt

```diff
@@ -79,8 +79,14 @@ class PlayerTOCAdapter(
     UIThread.checkIsUIThread()
 
     val item = this.spineElements[position]
+    val normalIndex = item.index + 1
```

Comment:
Minor point: It might be best to use `item.chapter` as the number.

Reply:
That brings up a question: `item.chapter` is set to be the same as `item.index`: https://github.com/NYPL-Simplified/audiobook-android/blob/ea72bba4b24971421ec8c3ba2246364c00ab4cbe/org.librarysimplified.audiobook.open_access/src/main/java/org/librarysimplified/audiobook/open_access/ExoManifest.kt#L56-L64

Is it supposed to be set to `index + 1` there? Or should I add 1 to it for display, like I'm doing with `item.index`?

Reply:
Good catch.

That is a good point... I think the meanings of those two fields have gotten a little fuzzy, looking at the various modules we have. I think `part` is always zero for anything that isn't a Findaway audiobook. For open access books, we treat `chapter == index` and then add `1` to it for display (because that looks more natural for non-programmers).

The `index` value, not the `chapter` value, is used to locate files on disk:

https://github.com/NYPL-Simplified/audiobook-android/blob/develop/org.librarysimplified.audiobook.open_access/src/main/java/org/librarysimplified/audiobook/open_access/ExoAudioBook.kt#L165

I can't be certain that we don't refer to `chapter` or `part` for anything other than display purposes, so I can't be certain that either adding `1` or not adding `1` is the correct behaviour for all possible backends...

I think perhaps just leave your code as-is, and I'll do some refactoring after this release to ensure we always treat the values the same way in all contexts.